pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1196,6 +1196,17 @@ status:
     kind: TaskRun
     name: test-pipeline-missing-results-task1
     pipelineTaskName: task1
+  provenance:
```

Comment:
please consider introducing `ignoreProvenance` similar to - https://github.com/tektoncd/pipeline/pull/7091/files#diff-a64a19b939f791920d235cbf5bf2e4905d4814af06f21401b27af11351afbd0bR1214

This provenance will no longer be needed with that change